evm/src/executor/inspector/cheatcodes/util.rs

```diff
@@ -26,6 +27,13 @@ fn addr(private_key: U256) -> Result<Bytes, Bytes> {
         return Err("Private key cannot be 0.".to_string().encode().into())
     }
 
+    let secp256k1_order =
+        U256::from_str("0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364141")
+            .unwrap();
+    if private_key > secp256k1_order {
+        return Err("Private key is greater than secp256k1 curve order.".to_string().encode().into())
```

Comment:
I suspect most user's won't know what "curve order" means or what that value is. What do you think about: 
```suggestion
        return Err("Private key must be less than 115792089237316195423570985008687907852837564279074904382605163141518161494337.".to_string().encode().into())
```